test/aarch64/run_suite.py

```diff
@@ -0,0 +1,78 @@
+from scripts.basic_test import BasicTest
+from scripts.basic_test_pie import BasicTestPie
+from scripts.basic_test_hw import BasicTestHw
+from scripts.breakpoint_test import BreakpointTest
+from scripts.brute_test import BruteTest
+from scripts.builtin_handler_test import BuiltinHandlerTest
+from scripts.callback_test import CallbackTest
+from scripts.catch_signal_test import CatchSignalTest
+from scripts.control_flow_test import ControlFlowTest
+from scripts.death_test import DeathTest
+from scripts.finish_test import FinishTest
+from scripts.floating_point_test import FloatingPointTest
+from scripts.handle_syscall_test import HandleSyscallTest
+from scripts.hijack_syscall_test import HijackSyscallTest
+from scripts.jumpstart_test import JumpstartTest
+from scripts.memory_test import MemoryTest
+from scripts.next_test import NextTest
+from scripts.signals_multithread_test import SignalMultithreadTest
+from scripts.speed_test import SpeedTest
+from scripts.thread_test_complex import ThreadTestComplex
+from scripts.thread_test import ThreadTest
+from scripts.watchpoint_test import WatchpointTest
+
+def fast_suite():
+    suite = TestSuite()
+
+    suite.addTest(TestLoader().loadTestsFromTestCase(AttachDetachTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(AutoWaitingTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(BacktraceTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(BasicTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(BasicTestPie))
+    suite.addTest(TestLoader().loadTestsFromTestCase(BasicTestHw))
+    suite.addTest(TestLoader().loadTestsFromTestCase(BreakpointTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(BruteTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(BuiltinHandlerTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(CallbackTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(CatchSignalTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(ControlFlowTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(DeathTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(FinishTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(FloatingPointTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(HandleSyscallTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(HijackSyscallTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(JumpstartTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(MemoryTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(NextTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(SignalMultithreadTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(SpeedTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(ThreadTestComplex))
+    suite.addTest(TestLoader().loadTestsFromTestCase(ThreadTest))
+    suite.addTest(TestLoader().loadTestsFromTestCase(WatchpointTest))
+
+    return suite
+
+
+if __name__ == "__main__":
```

Comment:
Why for arm we don't have the thread_stress option?

Reply:
Thread behavior does not depend on the architecture, so I didn't feel the need to bring it over.
Furthermore, I will be working on a refactor of the test suite next, which would solve this issue.

Reply:
OK, I agree on the refactor. I was about to open an issue on that, but if you are already working on it, I won't proceed. Please, make sure to open issues or write on the projects that you are working on or plan to work on next. After the holidays, we need to have a clear idea of who is doing what to avoid overlaps and conflicts. :)